apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/core/kafka/KafkaProducerManager.java

```diff
@@ -142,16 +142,6 @@ private void notifyListeners(KafkaConnectionStatus status) {
         }
     }
 
-    private void closeAdminClient(AdminClient adminClient) {
-        if (adminClient != null) {
-            try {
-                adminClient.close();
-            } catch (Exception e) {
-                LOGGER.error("close kafka admin client failed", e);
-            }
-        }
-    }
-
```

Comment:
What happened? Why removing this? Read #7016. If you want any concerns, we should loop @darcydai in.

Reply:
`adminClient.close()` has moved to L121  
Directly close before the operation.

Reply:
OK, you need to make CI passed.